validator/sawtooth_validator/journal/consensus/consensus.py

```diff
@@ -38,6 +38,8 @@ def __init__(self, block_cache, state_view, batch_publisher):
                 batch_publisher: An interface implementing send(txn_list)
                 which wrap the transactions in a batch and broadcast that
                 batch to the network.
+                data_dir: path to location where persistant data for the
```

Comment:
Interestingly I am no the only one to make that spelling error:
C:\Users\clmontgo\Workspaces\sawtooth\sawtooth-virtual-env-win\Lib\site-packages\twisted\persisted\sob.py:
  140          log.msg("Saved.")
  141  
  142: # "Persistant" has been present since 1.0.7, so retain it for compatibility
  143: Persistant = Persistent
  144  
  145  class _EverythingEphemeral(styles.Ephemeral):
  ...
  238      }[ext]
  239  
  240: __all__ = ['loadValueFromFile', 'load', 'Persistent', 'Persistant',
  241             'IPersistable', 'guessType']
  242  

C:\Users\clmontgo\Workspaces\sawtooth\sawtooth-virtual-env-win\Lib\site-packages\win32\lib\win32serviceutil.py:
  259              win32api.RegCloseKey(key)
  260  
  261: # Utility functions for Services, to allow persistant properties.
  262  def SetServiceCustomOption(serviceName, option, value):
  263      try:

C:\Users\clmontgo\Workspaces\sawtooth\sawtooth-virtual-env-win\Lib\site-packages\win32com\client\gencache.py:
    8    Each typelib is generated into a filename of format "{guid}x{lcid}x{major}x{minor}.py"
    9    
   10:   An external persistant dictionary maps from all known IIDs in all known type libraries
   11    to the type library itself.